control

```diff
@@ -12,4 +12,4 @@ XB-UserVisible: yes
 Section: Add-Ons
 XB-DisplayVersion: {display_version}
 XB-DisplayName: VeriStand Custom Device Wizard for LabVIEW {labview_version}
-Depends: ni-labview-{labview_version}{pkg_x86_bitness_suffix} (>= {labview_short_version}.0.0), ni-veristand-{labview_version} (>= {labview_short_version}.0.0), ni-custom-device-development-tools-{labview_version}-support (>= 23.0.0)
+Depends: ni-labview-{labview_version}{pkg_x86_bitness_suffix} (>= {labview_short_version}.0.0), ni-veristand-{labview_version} (>= {labview_short_version}.0.0), ni-custom-device-development-tools-{labview_version}-support (>= 24.0.0)
```

Comment:
> should we use {labview_short_version} or similar here instead of hard coding it?

There was some reason Ryan used hard-coded values instead of variables, but I can't track down what that reasoning is. It may be that isn't a problem anymore, but the other repos still have hard-coded versions, so I'm going to leave it for now.